public/static/docs/command-reference/get.md

```diff
@@ -58,13 +58,17 @@ name.
   an existing directory is specified, then the output will be placed inside of
   it.
 
-- `--rev` - `url` is expected to represent a Git repository for this option to
+- `--rev` - `url` is expected to contain a Git repository for this option to
   have an effect. Specific
   [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
   (such as a branch name, a tag, or a commit hash) of the repository to download
   the file or directory from. The tip of the default branch is used by default
   when this option is not specified.
 
+- `--show-url` - `path` is expected to represent a cached, DVC-tracked file for
```

Comment:
same here, still would prefer to see the essence of it. After that something like: Limitations: it does not print anything (or throws an exception?) for data files or directories that are not DVC-tracked (usually, saved into Git directly instead). See example ... link"

Reply:
OK, moved the prereq to last in cab347ca.

Reply:
And made standard use of this order in other commands.

Reply:
> Limitations: it does not print anything...

p.s. I think it's best to just say what the prereq is and not what the command will do if the prereq is not met. Easier text to maintain.